pages/common/sbuild.md

```diff
@@ -0,0 +1,32 @@
+# sbuild
+
+> Build a Debian binary package in a clean `chroot` environment.
+> More information: <https://wiki.debian.org/sbuild>.
+
+- Build the package in the current directory:
+
+`sbuild`
+
+- Build the given package:
+
+`sbuild {{package}}`
+
+- Build for a certain distribution:
+
+`sbuild --dist {{distribution}}`
+
+- Build using custom dependencies:
+
+`sbuild --extra-package {{path/to/deb_files}}`
```

Comment:
This sounds like the parameter should be a directory, but the documentation suggests otherwise:
> use the `--extra-package=./foo.deb` option

```suggestion
`sbuild --extra-package {{path/to/file.deb}}`
```

Reply:
It can be both in fact, and the example I would like to provide is passing a directory.

Reply:
--extra-package=package.deb|directory
              Make package.deb available for build-dependency resolution, by adding it to a temporary  archive  cre‐
              ated  by  sbuild.   This  makes  it easier to build packages against locally-built build dependencies,
              without waiting for those packages to enter the main archive, or going through the hassle of maintain‐
              ing a local archive and making it accessible inside the chroot.  package.deb is copied  into  the  ch‐
              root,  so  it can refer to any path on the host system.  If a directory is passed instead of a regular
              file, then all regular files inside that directory with a filename that ends in .deb will be added  in
              the  same fashion as it is done for individual packages.